### PR TITLE
[Earlgrey] Settings' suite expanded

### PR DESCRIPTION
**Pull request by jesmrec**
Polish the Settings suite by adding checks over the latest settings: Display settings, Media upload, and changes in Logging options.

### CODE REVIEW

File: ownCloudTests/SettingsTests.swift [hunk 1]
```diff
@@ -31,6 +31,32 @@ class SettingsTests: XCTestCase {
...
+	/*
+	* PASSED if: Show hidden files and folders are displayed as part of the "Display Settings" section of Settings
+	*/
+	func testCheckDisplaySettings () {
+		
+		//Assert
+		EarlGrey.select(elementWithMatcher: grey_accessibilityID("show-hidden-files-switch")).assert(grey_sufficientlyVisible())
+		
+		//Reset status
+		EarlGrey.select(elementWithMatcher: grey_text("ownCloud".localized)).perform(grey_tap())
```
Comment (hosy) [inline ownCloudTests/SettingsTests.swift]: Please could you replace "ownCloud" with `OCAppIdentity.shared.appName`?

File: ownCloudTests/SettingsTests.swift [hunk 2]
```diff
@@ -31,6 +31,32 @@ class SettingsTests: XCTestCase {
...
+	* PASSED if: Media upload options are displayed as part of the "Media Upload" section of Settings
+	*/
+	func testCheckMediaUploadSettings () {
+		
+		//Assert
+		EarlGrey.select(elementWithMatcher: grey_accessibilityID("convert_heic_to_jpeg")).assert(grey_sufficientlyVisible())
+		EarlGrey.select(elementWithMatcher: grey_accessibilityID("convert_to_mp4")).assert(grey_sufficientlyVisible())
+		
+		//Reset status
+		EarlGrey.select(elementWithMatcher: grey_text("ownCloud".localized)).perform(grey_tap())
```
Comment (hosy) [inline ownCloudTests/SettingsTests.swift]: Please could you replace "ownCloud" with `OCAppIdentity.shared.appName`?

File: ownCloudTests/SettingsTests.swift [hunk 3]
```diff
@@ -83,20 +109,31 @@ class SettingsTests: XCTestCase {
...
+		//Actions
+		EarlGrey.select(elementWithMatcher: grey_accessibilityID("logging")).perform(grey_tap())
+		EarlGrey.select(elementWithMatcher: grey_text("Warning".localized)).perform(grey_tap())
+		EarlGrey.select(elementWithMatcher: grey_text("Settings".localized)).perform(grey_tap())
+
+		//Assert
+		EarlGrey.select(elementWithMatcher: grey_text("Warning".localized)).assert(grey_sufficientlyVisible())
+
+		//Reset status
+		EarlGrey.select(elementWithMatcher: grey_text("ownCloud".localized)).perform(grey_tap())
```
Comment (hosy) [inline ownCloudTests/SettingsTests.swift]: Please could you replace "ownCloud" with `OCAppIdentity.shared.appName`?

File: ownCloudTests/SettingsTests.swift (hunk 1 above)
Comment (jesmrec) [reply ownCloudTests/SettingsTests.swift]: 👍

File: ownCloudTests/SettingsTests.swift (hunk 2 above)
Comment (jesmrec) [reply ownCloudTests/SettingsTests.swift]: 👍

File: ownCloudTests/SettingsTests.swift (hunk 3 above)
Comment (jesmrec) [reply ownCloudTests/SettingsTests.swift]: 👍

**Comment by jesmrec** [comment]
@hosy changes done.

**Pull request merged**